components/org.wso2.carbon.database.utils/src/main/java/org/wso2/carbon/database/utils/jdbc/NamedPreparedStatement.java

```diff
@@ -214,6 +214,20 @@ public void setNString(String name, String value) throws SQLException {
         }
     }
 
+    /**
+     * Sets the designated parameter to the given input stream, which will have the specified number of bytes.
+     *
+     * @param name Name of the index.
+     * @param inputStream The java input stream which contains the binary parameter value.
+     * @param length The number of bytes in the stream.
+     * @throws SQLException if name does not correspond to a parameter marker in the SQL statement; if a database access
+     * error occurs or this method is called on a closed {@code PreparedStatement}.
+     */
```

Comment:
```suggestion
    /**
     * Sets the designated parameter to the given input stream, which will have the specified number of bytes.
     *
     * @param name        Name of the index.
     * @param inputStream The java input stream which contains the binary parameter value.
     * @param length      The number of bytes in the stream.
     * @throws SQLException If name does not correspond to a parameter marker in the SQL statement; if a database access
     *                      error occurs or this method is called on a closed {@code PreparedStatement}.
     */
```

Let's format the method comment.

Reply:
Fixed with https://github.com/wso2/carbon-utils/compare/c2a800d2a8793629b7f2b3c8cb91dbe70178f598..90b5d4300fafc5d4abaaadb7e41a9a340429f390